examples/README.adoc

```diff
@@ -0,0 +1,21 @@
+= TomEE Examples
+
+These examples demonstrate particular features of the JEE ecosystem to help
```

Comment:
JEE should be Java EE
JEE is not an authorized acronyme.

Reply:
Thanks, Done